Sources/Algorithms/Partition.swift

```diff
@@ -210,8 +210,9 @@ extension Collection {
 //===----------------------------------------------------------------------===//
 
 extension Sequence {
-  /// Returns two arrays containing, in order, the elements of the sequence that
-  /// do and don’t satisfy the given predicate.
+  /// Returns two arrays containing the elements of the sequence that
+  /// don’t and do satisfy the given predicate, respectively. The order of the
```

Comment:
“The order of the arrays” makes it seem like the order of the tuple’s elements. Perhaps “in” should be used here?

```suggestion
  /// don’t and do satisfy the given predicate, respectively. The order in the
```

Reply:
Adjusted the wording a bit, good point!